README.md

```diff
@@ -19,6 +19,8 @@ that run on the Ethereum Virtual Machine. Smart contracts are programs that are
 network where nobody has special authority over the execution and thus they allow to implement tokens of value,
 ownership, voting and other kinds of logics.
 
+When deploying contracts, you should use the latest released version of Solidity. This is because breaking changes as well as new features and bug fixes are introduced regularly. We currently use a 0.x version number [to indicate this fast pace of change](https://semver.org/#spec-item-4).
```

Comment:
I think this may slightly misleading to say that if the major number would be 1, then bumping it wouldn't mean a breaking change. Though I understand where you're coming from, being in major number 0 suggests changes are more frequent.

Reply:
I intentionally left out any mention of what happens at >= 1.0 so as not to make commitments on your behalf. But presumably, we would follow normal semver practices at that time and update the statement here.